src/gui/sharelinkwidget.cpp

```diff
@@ -435,6 +478,21 @@ void ShareLinkWidget::slotAnimationFinished()
     deleteLater();
 }
 
+void ShareLinkWidget::slotCreateLabel()
+{
+    if (_linkShare) {
+        slotToggleButtonAnimation(_shareLinkButton, _shareLinkProgressIndicator, true, true);
```

Comment:
not related to this PR but this `slotToggleButtonAnimation` method could use some improvements to avoid bool hell
no way I can predict what those `true, true` or `true, false` parameters will do

Reply:
Ok, I started on it... it is a black hole to refactor it... 
I will create another PR for it later.